tests/Sniffs/Attributes/data/disallowedAttributeJoiningNoErrors.php

```diff
@@ -0,0 +1,20 @@
+<?php // lint >= 8.0
+
+#[Attribute1] #[Attribute2('var')]
+#[Attribute3(option: PDO::class, option2: true, option3: 'False')]
+class TestClass
+{
+    public function __construct(#[Attribute1] #[Attribute2('var')]
+                                #[Attribute3(option: PDO::class, option2: true, option3: 'False')] $test)
+    {
+    }
+}
+
+$object = new #[Attribute1] #[Attribute2('var')]
+    #[Attribute3(option: PDO::class, option2: true, option3: 'False')] class () { /* … */ };
+
+#[Attribute1] #[Attribute2('var')]
+#[Attribute3(option: PDO::class, option2: true, option3: 'False')]
+function testFunc($test)
+{
+}
```

Comment:
Please test all other missing types, eg. closure, constant or property.

Reply:
@michnovka There is "eg." in my comment. All `attributableTokens` should be checked.

You should also check that only parameters and properties are ok for `T_VARIABLE`.

Reply:
@kukulich  hmm, since I added enums into the data sample, lint for 8.0 is failing (needs 8.1). How do we approach this? Do I create special test case for enums with lint target 8.1 and keep the rest under 8.0, or do I just bump the tests to 8.1 for the whole file?

Reply:
> do I just bump the tests to 8.1 for the whole file?

Just bump it for the whole file.

Reply:
Ok I have added an example of invalid attribute and its treated as a comment. And not fixed.